### PR TITLE
(BKR-1032) beaker init UX improvements

### PR DESCRIPTION
**Pull request by ferglor**

### CODE REVIEW

**Comment by ferglor** [comment]
@puppetlabs-jenkins retest this please

**Comment by ferglor** [comment]
@puppetlabs-jenkins retest this please

**Comment by ferglor** [comment]
@puppetlabs-jenkins retest this please

**Comment by ferglor** [comment]
@puppetlabs-jenkins retest this please

**Comment by tvpartytonight** [comment]
@puppetlabs-jenkins retest this please

**Comment by ferglor** [comment]
@puppetlabs-jenkins retest this please

File: lib/beaker/subcommands/subcommand_util.rb [hunk 1]
```diff
@@ -110,6 +103,20 @@ def self.init_hypervisor(options)
           init_vmpooler
         end
       end
+
+      # Execute a task but capture stdout and stderr to a buffer
+      def self.with_captured_output
+        begin
+          old_stdout = $stdout.clone
+          old_stderr = $stderr.clone
+          $stdout = StringIO.new
```
Comment (tvpartytonight) [inline lib/beaker/subcommands/subcommand_util.rb]: Is there a chance we might run into memory issues with this for extremely long test runs, or test runs with a lot of output? We should be wary of that when we are implementing `beaker run`.

**Comment by tvpartytonight** [comment]
LGTM 👍

**Pull request merged**